hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/resolver/order/AvailableSpaceResolver.java

```diff
@@ -118,6 +121,20 @@ protected String chooseFirstNamespace(String path, PathLocation loc) {
         getSubclusterMapping();
     List<SubclusterAvailableSpace> subclusterList = new LinkedList<>(
         subclusterInfo.values());
+
+    if (loc != null && loc.getDestinations() != null) {
```

Comment:
Hi, @wzk784533 I think this modification is good, but I want to consult under what circumstances this loc or loc.getDestinations() would be null.